include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -710,11 +718,14 @@ Operator<dim, Number>::compute_initial_acceleration(VectorType &       accelerat
   VectorType rhs(acceleration);
   rhs = 0.0;
 
+  double scaling_factor_mass;
```

Comment:
No need to define this outside. We can simply write `double const` inside the if-branch and the else-branch. Makes the code easier to read in my opinion (and in the sense of @btemuer, the use of `scaling_factor_mass` is here again in a `const` way).

Reply:
NB means "nota bene" (something like "Please note that ...")